README.md

```diff
@@ -0,0 +1,4 @@
+Becklyn Cache Bundle
+====================
+
+Provides a simple cache item, that should be easy to use and high performance.
```

Comment:
Could you provide a brief example which interface to implement, possible with a small example snippet?

Reply:
Added in the README